lib/sdc-events.js

```diff
@@ -1032,7 +1032,9 @@ function main() {
         function getNodes(ctx, next) {
             CNAPI.listServers(function (err, servers) {
                 ctx.nodes = servers.filter(function (server) {
-                    return server.status === 'running' && server.setup;
+                    return server.status === 'running' && server.setup &&
+                            // Ignore virtual servers.
+                            (server.hostname || '').substr(0, 2) !== 'VC';
```

Comment:
That "VC" is just a convention. A better differentiator is: `server.sysinfo["System Type"] !== "Virtual"` if you are able to use that.

Reply:
E.g. see the changes in https://jira.joyent.us/browse/TOOLS-2271

Reply:
Thanks - fixed.